packages/core/src/compiler/Style.ts

```diff
@@ -2565,7 +2590,9 @@ const evalAccess = (
     case "ColorV":
     case "FloatV":
     case "PathDataV":
-    case "StrV": {
+    case "StrV":
+    case "ShapeListV": {
+      // Not allowing indexing into a shape list for now
```

Comment:
Maybe it would be better to make a new error type for this? The error message for `NotCollError` says "Cannot index into a non-collection", which would be somewhat confusing in this case.

Reply:
Done